.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+updates:
+- package-ecosystem: github-actions
+  directory: "/"
+  schedule:
+    interval: monthly
```

Comment:
nit: maybe we would be ok to have this more frequently?

Reply:
I configured it the same way like the core repo, we can extend the cadence if needed.